src/pages/publish.tsx

```diff
@@ -4,7 +4,7 @@ import Page from '../components/templates/Page'
 import { graphql, PageProps } from 'gatsby'
 
 export default function PageGatsbyPublish(props: PageProps): ReactElement {
-  const content = (props.data as any).content.edges[0].node.childPagesJson
+  const content = (props.data as any).content.edges[0].node.childPublishJson
```

Comment:
Nice 😃 , I also initially tried to move them into publish folder but the query didn't really work, but I see now why 👍

Reply:
yes, this is super confusing that Gatsby picks up the folder names to construct the query structure